posts/2021-09-03-210010-beta.adoc

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+title: "Java 17 beta support, MicroProfile Context Propagation 1.3, and other enhancements in Open Liberty 21.0.0.10-beta"
```

Comment:
I would change this, and all other references, of `Java 17 beta support` to `Java 17 early access support` since that is how Oracle refers to their releases still under development.